satpy/tests/reader_tests/test_slstr_l2.py

```diff
@@ -19,9 +19,10 @@
 
 import unittest
 from unittest import mock
-from unittest.mock import MagicMock
-from unittest.mock import patch
+from unittest.mock import MagicMock, patch
```

Comment:
Here it changed multiple imports to a single one, but elsewhere (see above) it did the exact opposite.  What is the rule?